widgetastic_manageiq/__init__.py

```diff
@@ -974,6 +974,50 @@ def read(self):
         return {field: self.get_text_of(field) for field in self.fields}
 
 
+class NestedSummaryTable(SummaryTable):
+    HEADER_IN_ROWS = './tbody/tr[1]/td'
+    HEADERS = './tbody/tr[1]/td/strong'
+
+    def __init__(self, parent, title, *args, **kwargs):
+        SummaryTable.__init__(self, parent, title, *args, **kwargs)
+
+    def _all_rows(self):
+        for row_pos in range(1, len(self.browser.elements(self.ROWS, parent=self))):
+            yield self.Row(self, row_pos)
+
+    def read(self):
+        return [{key: col.text for key, col in row} for row in self]
+
+
+class StatusBox(Widget, ClickableMixin):
+    LOC = ("//span[contains(@class,'card-pf-aggregate-status-count')]"
```

Comment:
REQUIRED: please use ParametrizedLocator/ParametrizedView instead. 
and I guess this xpath has to be reworked.

@mfalesni, please advice better implementation of this widget

Reply:
mfalesni is on sick leave today. So, let's discuss this widget maybe we could make up something better.